build.gradle

```diff
@@ -132,9 +132,9 @@ dependencies {
 
     compile 'org.eclipse.jetty:jetty-servlets:9.2.14.v20151106'
 
-    providedCompile 'org.springframework.boot:spring-boot-starter-tomcat:1.3.5.RELEASE'
+    compile 'org.springframework.boot:spring-boot-starter-tomcat:1.3.5.RELEASE'
 
-    providedRuntime 'org.hsqldb:hsqldb:2.3.4'
+    runtime 'org.hsqldb:hsqldb:2.3.4'
```

Comment:
Same as before. In addition, this version is not the same as the one used in the Scheduling project which may create issues.